srcs/utils/count_line_len.c

```diff
@@ -0,0 +1,22 @@
+#include <utils.h>
+
+int	get_max_line_len(const char **map)
```

Comment:
Looking at this, I'm not sure if we have to create a struct for the lines length. What do you think?